File: periode 1/8-10-2018/pset6/mario/less/mario.py
from cs50 import get_int

while True:
    height = get_int("height: ")
    if height > 0 and height < 24:
        break
for i in range(height):
    print(" " * (height - 1 - i), end="")
    print("#" * (2 + i), end="")
    print()

File: periode 1/8-10-2018/pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>
long long getal;


int main(void)
{
    int h;
    do
    {
        h = get_int("Height:");
    }
    while (h < 0 || h > 23);

    for (int i = 0; i < h; i++)
    {
        for (int s = i; s < h - 1; s++)
        {
            printf(" ");
            //print spaces
        }
        for (int a = 0; a < i + 1; a++)
        {
            printf("#");
            //print hashtags
        }
        printf("  ");
        for (int n = 0; n < i + 1; n++)
        {
            printf("#");
        }
        printf("\n");
        //print enter
    }
}

